src/components/Navigation.js

```diff
@@ -169,22 +213,196 @@ function Navigation() {
           <div className="d-flex flex-row align-items-center ms-auto navigation-search">
             <div className="d-flex flex-row align-items-center">
               <input
-                className="form-control me-2"
+                className="form-control me-2 bg-dark text-light navigation-search-input"
                 type="search"
-                placeholder="Search tx or address"
+                placeholder={`Search here`}
```

Comment:
issue: The UI is not indicative of what can be searched.

We should keep the "Search tx or address" so users know that he can also search for an address.
Or maybe we can keep the "Search here" and when the user focus on the input it shows a message explaining that he can search both txs and addresses.

Reply:
We changed it accordingly with the figma prototype. 

![Screenshot_20241021_171735_Firefox](https://github.com/user-attachments/assets/19edb10d-2c83-4c49-84f0-28823fdc2a0c)

Should we change it back?

Reply:
> We changed it accordingly with the figma prototype.

You're right, I think we can merge like this and we can talk with Ana later since this change is simple to rollback.